.github/workflows/test-indexer-debian.yml

```diff
@@ -0,0 +1,42 @@
+name: Test Wazuh indexer on upgrade - Debian
+on:
+  push:
+    branches:
+      - 1850-wazuh-indexer-preserve-config-files-upon-upgrade-master
```

Comment:
This should not be hardcoded to a development branch, if testing is over, remember to change it to a numbered branch.

Reply:
Done.